core/sdk-core/src/main/java/software/amazon/awssdk/core/RequestOverrideConfiguration.java

```diff
@@ -46,6 +47,7 @@
     private final Duration apiCallAttemptTimeout;
     private final Signer signer;
     private final List<MetricPublisher> metricPublishers;
+    private final Map<ExecutionAttribute<?>, Object> executionAttributes;
```

Comment:
We should use [`AttributeMap`](https://github.com/aws/aws-sdk-java-v2/blob/master/utils/src/main/java/software/amazon/awssdk/utils/AttributeMap.java)